config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,19 @@
+# This patch add annotation to admission webhook config and
+# the variables $(CERTIFICATE_NAMESPACE) and $(CERTIFICATE_NAME) will be substituted by kustomize.
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: inferenceservice.serving.kubeflow.org
+  annotations:
+    cert-manager.io/inject-ca-from: kfserving-system/serving-cert
```

Comment:
Will this work with a manual cert if cert-manager isn't installed?

Reply:
I am not sure how to do this, the certs needs to be injected to the mutating/validating webhook configuration, since controller runtime no longer creates and fresh the cert, somewhere needs to do it, either cert manager or kfserving controller. I do not think we want to do this in kfserving controller, so I feel better solution is still use cert manager...  WDYT?

Reply:
belive kfserving would be installing its own cert-manger ?

Reply:
Kubeflow would have its own certmanager which we would use I believe. The easiest solution is to add self-signed certificates in the webhook which work and which certmanager will overwrite if it exists - it looks at the annotations. If certmanager doesn't exist you would need to allow users to not install the Certificate resources. or do we assume cert-manager for now?

Reply:
I am assuming cert-manager now, for a production grade kfserving installation I think this should be the preferred way, we just got bite on prod for expired certificates.